dev/com.ibm.ws.org.hibernate.validator.cdi/bnd.overrides

```diff
@@ -10,9 +10,12 @@ Fragment-Host:
 Import-Package: \
   org.apache.el,\
   *
-  
+
+Export-Package: org.hibernate.validator.cdi;version="6.0.4.Final";uses:="javax.enterprise.event,javax.enterprise.inject.spi,javax.inject,javax.validation,javax.validation.executable",\
+  org.hibernate.validator.cdi.internal;version="6.0.4.Final", \
+  org.hibernate.validator.cdi.internal.interceptor;version="6.0.4.Final"
+
 Private-Package: \
-  org.hibernate.validator.cdi.internal.*,\
```

Comment:
this line shouldn't be getting removed... that imples we're just throwing away all of the hibernate-validator-cdi code?